assets/src/application/ui/layout/entityList/pagination/EntityPagination.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { Pagination } from 'antd';
+import { PaginationProps } from 'antd/lib/pagination';
+
+import './style.scss';
+
+/**
+ * Adds pagination to an "EntityList" component
+ * @return EntityPagination
+ */
+const EntityPagination = ({
+	current,
+	onChange,
+	pageSize = 10,
+	pageSizeOptions = ['10', '20', '30', '40'],
```

Comment:
I know it's unconventional, but can we use the same options as EDTR v1 plz? ie: 6, 12, 24, 48
those are all more divisable than multiples of 10.
for example, other than 6, the other numbers (12, 24, & 48) can all divide neatly into 2, 3, or 4 columns, whereas 10s can only fit into 2 or 5

Reply:
Thx for explanation, updated: https://github.com/eventespresso/event-espresso-core/pull/2372/commits/67d37e6a72c897ec6f13e5404fbf4d19951d525b